packages/components/drawer/src/token.ts

```diff
@@ -0,0 +1,13 @@
+import type { InjectionKey, Ref } from 'vue'
```

Comment:
Maybe we shoud move this file to `@element-plus/token`

Reply:
We can do that later in another PR for this.